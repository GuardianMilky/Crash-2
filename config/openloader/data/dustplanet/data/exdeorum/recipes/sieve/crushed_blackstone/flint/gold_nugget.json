{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:crushed_blackstone"},"mesh":"exdeorum:flint_mesh","result":"minecraft:gold_nugget","result_amount":{"type":"minecraft:binomial","n":4.0,"p":0.225}}