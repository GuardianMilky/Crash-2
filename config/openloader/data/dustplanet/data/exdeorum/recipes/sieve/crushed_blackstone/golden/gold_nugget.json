{"type":"exdeorum:sieve","ingredient":{"item":"exdeorum:crushed_blackstone"},"mesh":"exdeorum:golden_mesh","result":"minecraft:gold_nugget","result_amount":{"type":"minecraft:binomial","n":8.0,"p":0.325}}